Makefile

```
@@ -1,59 +1,57 @@
-.PHONY: build
+include .env
+
+.PHONY: *
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)